idea-plugin/src/main/kotlin/arrow/meta/ide/testing/env/IdeTestInterpreter.kt

```diff
@@ -0,0 +1,25 @@
+package arrow.meta.ide.testing.env
+
+import arrow.meta.ide.testing.IdeTest
+
+fun assertThis(ideTest: IdeTest): Unit = interpreter(ideTest)
+
+val interpreter: (IdeTest) -> Unit = {
```

Comment:
So this does nothing?

Reply:
This is still WIP 😄 
But I am making progress.